### PR TITLE
Feat: update favicons

### PR DESCRIPTION
**Pull request by VictorVelarde**
To improve contrast over dark themes

### CODE REVIEW

File: template-skeleton/template/public/favicon/browserconfig.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <browserconfig>
     <msapplication>
         <tile>
-            <square150x150logo src="/favicon/mstile-150x150.png"/>
+            <square150x150logo src="/mstile-150x150.png"/>
```
Comment (neokore) [inline template-skeleton/template/public/favicon/browserconfig.xml:5]: This file is placed inside the `favicon` folder, is it right?

**Comment by neokore** [comment]
I've seen that all files have execute permissions, is it a MacOS thing?

File: template-skeleton/template/public/favicon/browserconfig.xml (hunk 1 above)
Comment (VictorVelarde) [reply template-skeleton/template/public/favicon/browserconfig.xml]: Correct, fixed

**Comment by VictorVelarde** [comment]
> I've seen that all files have execute permissions, is it a MacOS thing? I did nothing special on them, just got the files from design's dropbox... 🤷‍♂️

**Review by neokore — APPROVED** [review]
:ok:

**Pull request merged**